src/ui/List/OrderedList.stories.js

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import { List, Container } from 'semantic-ui-react';
+
+export default {
+  title: 'Components/List/Ordered List',
+  component: List,
+  argTypes: {
+    animated: {
+      description:
+        'a list can animate to set the current item apart from the list',
+      table: {
+        type: { summary: 'boolean' },
+        defaultValue: { summary: false },
+      },
+    },
+    divided: {
+      description: 'a list can show divisions between conten',
+      table: {
+        type: { summary: 'boolean' },
+        defaultValue: { summary: false },
+      },
+    },
+    items: {
+      description: 'array of list content',
+      table: {
+        type: { summary: 'object' },
+        defaultValue: { summary: ' "" ' },
+      },
+    },
+  },
+};
+
+function OrderedList(items, animated, divided, horizontal) {
+  return (
+    <Container>
+      <List as="ul" ordered animated={animated} divided={divided}>
```

Comment:
@g-stamatis All list should use HTML semantic elements for listing. Thus, here we should use:

```
<ol>
  <li></li>
</ol>
```